test/cpp/api/dataloader.cpp

```diff
@@ -23,6 +23,7 @@
 #include <vector>
 
 using namespace torch::data; // NOLINT
+using namespace c10;
```

Comment:
I think we decided not to have this directive. In fact I think it has no effect since you're using `torch::optional` anyway.

Reply:
I thought I did too. fixed.